book/api/websocket.md

```diff
@@ -586,21 +586,21 @@ potential underflow.
 | Field               | Type                | Description
 |---------------------|---------------------|------------
 | next_leader_slot    | `number\|null`      | The next leader slot |
-| tile_primary_metric | `TilePrimaryMetric` | Per-tile-type primary metrics.  Some of these are point-in-time values (P), and some are aggregated since the end of the previous leader slot (A) |
+| tile_primary_metric | `TilePrimaryMetric` | Per-tile-type primary metrics.  Some of these are point-in-time values (P), and some are moving window averages (W) |
```

Comment:
Mention that it's 1s moving average

Reply:
Also is there a separate part of the docs clarifying that it's the max when querying a historical slot?

Reply:
Yes, in the doc for SlotResponse